src/nnbench/runner.py

```diff
@@ -23,6 +23,12 @@ def _check(params: dict[str, Any], benchmarks: list[Benchmark]) -> None:
     for bm in benchmarks:
         for name, param in inspect.signature(bm.fn).parameters.items():
             param_type = param.annotation
+
+            if param.annotation == inspect.Parameter.empty:
+                logger.warn(
+                    f"Warning: Parameter {name!r} in benchmark '{bm.fn.__name__}' is untyped."
+                )
```

Comment:
I think this should be a debug log - it's more of a convenience for the user if they want to understand why something is failing, and we will always and unconditionally advertise typing benchmarks completely.

```suggestion
            if param_type == inspect.Parameter.empty:
                logger.debug(
                    f"Found untyped parameter {name!r} in benchmark {bm.fn.__name__!r}."
                )
```